pkg/storage/bloom/v1/builder_test.go

```diff
@@ -226,7 +226,7 @@ func TestMergeBuilder(t *testing.T) {
 	)
 
 	// Ensure that the merge builder combines all the blocks correctly
-	mergeBuilder := NewMergeBuilder(dedupedBlocks(blocks), storeItr, pop)
+	mergeBuilder := NewMergeBuilder(dedupedBlocks(blocks), storeItr, pop, NewMetrics(nil))
```

Comment:
Shouldn't a registerer be passed to `NewMetrics()`?

Reply:
it short circuits with nil